Library/Homebrew/sandbox.rb

```diff
@@ -95,12 +97,26 @@ def exec(*args)
     seatbelt.close
     @start = Time.now
 
+    $stdin.raw!
+    stdin_thread = T.let(nil, T.nilable(Thread))
+
     begin
-      T.unsafe(self).safe_system SANDBOX_EXEC, "-f", seatbelt.path, *args
+      command = [SANDBOX_EXEC, "-f", seatbelt.path, *args]
+      T.unsafe(PTY).spawn(*command) do |r, w, pid|
```

Comment:
```suggestion
      # Start sandbox in a pseudoterminal to prevent access of the parent terminal.
      T.unsafe(PTY).spawn(*command) do |r, w, pid|
```